Create lwm2m-agent-1021-7-1-fixed-a-rare-condition-where-already-expired-realtime-subscription-was-still.md (#3070)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-7-1-fixed-a-rare-condition-where-already-expired-realtime-subscription-was-still.md

---
date: ""

> title: Fixed a rare condition where already expired realtime subscription  was still considered active by LWM2M agent

Reviewer: ```suggestion
title: Fixed a rare condition where LWM2M device operation realtime subscription considered active for already expired subscription in LWM2M service
```

> title: Fixed a rare condition where LWM2M device operation realtime subscription considered active for already expired subscription in LWM2M service

Reviewer 2: ```suggestion
title: Fixed an issue with expired LWM2M device operation realtime subscriptions
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-4363
version: 1021.7.1
---

> In previous versions it was possible that expired realtime subscription was considered active by LWM2M agent. It could lead to issues like disconnecting from already disconnected channel or not being able to connect again. This issue is now fixed.

Reviewer: ```suggestion
In previous versions it was possible that expired realtime subscription was considered active by LWM2M agent. It could lead to issues like disconnecting from already disconnected channel or not being able to connect again. This caused the LWM2M device operations not taken in realtime for the device. This issue is now fixed.
```

> In previous versions it was possible that expired realtime subscription was considered active by LWM2M agent. It could lead to issues like disconnecting from already disconnected channel or not being able to connect again. This caused the LWM2M device operations not taken in realtime for the device. This issue is now fixed.

Reviewer 2: ```suggestion
In previous versions it was possible that an expired realtime subscription was considered active by the LWM2M agent. This could lead to issues like disconnecting from an already disconnected channel or not being able to connect again. As a result, the LWM2M device operations were not taken in realtime for the device. This issue has now been fixed.
```
